tern/analyze/default/container/multi_layer.py

```diff
@@ -73,12 +73,12 @@ def analyze_subsequent_layers(image_obj, shell, master_list, redo,
                               driver=None):
     """Assuming that we have a shell and have completed analyzing the first
     layer of the given image object, we now analyze the remaining layers.
-    While we have layers:
```

Comment:
This is unnecessary change in the comment, I'll remove it (previously I had to change logic here but now it works out iof the box due to only necessary layers being loaded into the image.layer[])